otto_connector/src/otto_connector/robot.py

```diff
@@ -42,8 +42,8 @@ def __init__(self, otto_id):
             InOrbitDataKeys.LAST_PLACE: None,  # {"name": "string", "id": "string"},
             InOrbitDataKeys.MISSION_TRACKING: {},  # Mission Tracking data
             InOrbitDataKeys.ONLINE_STATUS: False,  # Consider offline at startup until FM launches
-            InOrbitDataKeys.SYSTEM_STATE: None,  # string
-            InOrbitDataKeys.SUBSYSTEM_STATE: None,  # string
+            InOrbitDataKeys.SYSTEM_STATE_FULL: {},  # Current state records. List of Dictionaries
```

Comment:
I would rename this to `ROBOT_STATES` instead of `SYSTEM_STATE_FULL`. Reasonale is that the topic where both `system` and `sub-system` data are published is `robots/states`. WDYT?